### PR TITLE
Updating KB for retaining trace data

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Adding a complete description on tracing retention 

### Motivation

This item in the FAQ is out of date:
https://docs.datadoghq.com/tracing/faq/how-long-is-tracing-data-stored/

### Preview link

  
* https://docs-staging.datadoghq.com/gus/apm-trace-retention/tracing/faq/how-long-is-tracing-data-stored/

### CODE REVIEW

File: content/tracing/faq/how-long-is-tracing-data-stored.md [hunk 1]
```diff
@@ -3,4 +3,37 @@ title: How long is tracing data stored?
...
 ---
 
-Traces are stored for up to 6 months. The storage time for each trace is determined probabilistically. If you view a trace, it is guaranteed to be stored for 6 months.
\ No newline at end of file
+Individual traces are stored for up to 6 months.  
+
+To determine how long a particular trace will be stored, the Agent makes a sampling decision early in the trace's lifetime. In Datadog backend, sampled traces are retained according to time buckets:
+
+
+|Retention bucket|% of stream kept|
```
Comment (lejulie) [inline content/tracing/faq/how-long-is-tracing-data-stored.md]: I think we can remove the table; the example below feels clearer to me.

File: content/tracing/faq/how-long-is-tracing-data-stored.md [hunk 2]
```diff
@@ -3,4 +3,37 @@ title: How long is tracing data stored?
 kind: faq
 ---
 
-Traces are stored for up to 6 months. The storage time for each trace is determined probabilistically. If you view a trace, it is guaranteed to be stored for 6 months.
\ No newline at end of file
+Individual traces are stored for up to 6 months.  
+
+To determine how long a particular trace will be stored, the Agent makes a sampling decision early in the trace's lifetime. In Datadog backend, sampled traces are retained according to time buckets:
```
Comment (lejulie) [inline content/tracing/faq/how-long-is-tracing-data-stored.md]: I think we can merge this first line into this paragraph.

File: content/tracing/faq/how-long-is-tracing-data-stored.md (hunk 1 above)
Comment (l0k0ms) [reply content/tracing/faq/how-long-is-tracing-data-stored.md]: I would let it like this, even if the text below gives more info the table acts as a nice summary.

File: content/tracing/faq/how-long-is-tracing-data-stored.md (hunk 2 above)
Comment (l0k0ms) [reply content/tracing/faq/how-long-is-tracing-data-stored.md]: 👍

File: content/tracing/faq/how-long-is-tracing-data-stored.md [hunk 3]
```diff
@@ -3,4 +3,35 @@ title: How long is tracing data stored?
...
+For example, at `9:00am UTC Wed, 12/20` you would see:
+
+* **100%** of traces sampled on `Wed 12/20 03:00 - 09:00`
+* **25%** of traces sampled on `Wed 12/20 00:00` - `Wed 12/20 02:59`
+* **10%** of traces sampled on `Thurs 12/14 00:00` - `Tue 12/19 23:59`
+* **1%** of traces sampled on `7/1 00:00` - `12/13 23:59`
+* **0%** of traces before `7/1 00:00`
+
+
+Additionally, once a trace has been viewed, it continues to be available based on its trace ID `https://app.datadoghq.com/apm/trace/<trace_id>` even if it “expires” from the UI. This behavior is independent of the UI retention time buckets.
```
Comment (jeremy-lq) [inline content/tracing/faq/how-long-is-tracing-data-stored.md]: This reads to me as if these traces are retained indefinitely. Is that correct?

Comment (lejulie) [reply content/tracing/faq/how-long-is-tracing-data-stored.md]: As of now they are.  I don't think we need to say this explicitly though.

Comment (lejulie) [inline content/tracing/faq/how-long-is-tracing-data-stored.md]: This reads a little awkwardly to me.  Maybe "Additionally, once a trace has been viewed, it continues to be available by using its trace ID in the URL: ... (even if it "expires" from the UI)."

File: content/tracing/faq/how-long-is-tracing-data-stored.md [hunk 4]
```diff
@@ -3,4 +3,35 @@ title: How long is tracing data stored?
...
 
-Traces are stored for up to 6 months. The storage time for each trace is determined probabilistically. If you view a trace, it is guaranteed to be stored for 6 months.
\ No newline at end of file
+Individual traces are stored for up to 6 months but to determine how long a particular trace will be stored, the Agent makes a sampling decision early in the trace's lifetime. In Datadog backend, sampled traces are retained according to time buckets:
+
+
+|Retention bucket|% of stream kept|
+|:-----|:--------|
+|6 hours|100%|
+|current day (UTC time)|25%|
```
Comment (lejulie) [inline content/tracing/faq/how-long-is-tracing-data-stored.md]: This is tedious but can we capitalize "Current day" so it looks neater?

**Pull request merged**